asset/script.js

```diff
@@ -0,0 +1,253 @@
+const hamburger = document.querySelector('.mobile_menu');
+
+hamburger.addEventListener('click', () => {
+  const div = document.createElement('div');
+  const img = document.createElement('img');
+  const links = document.getElementsByClassName('nav_link')[0];
+  const home = document.getElementsByClassName('grid-wrapper')[0];
+  const divContainer = document.createElement('div');
+  const linksChilds = Array.from(links.children);
+  const property = {
+    color: '#fff',
+    fontSize: '32px',
+    fontFamily: '"Lato", sans-serif',
+    fontWeight: 'bold',
+    lineHeight: '44px',
+    textDecoration: 'none',
+  };
+
+  divContainer.style.backgroundColor = 'rgba(39,42,49,0.7)';
+  img.src = 'statics/images/Toolbar.png';
+  img.style.float = 'right';
+  img.style.marginRight = '1rem';
+  img.addEventListener('mouseover', () => {
+    img.style.cursor = 'pointer';
+  });
+  links.style.display = 'block';
+  links.style.clear = 'both';
+  divContainer.style.zIndex = '999';
+  divContainer.style.position = 'fixed';
+  divContainer.style.top = '0';
+  divContainer.style.height = '130vh';
+  divContainer.style.width = '100vw';
+  div.style.marginTop = '40px';
+  links.style.listStyle = 'none';
+  img.addEventListener('click', () => {
+    divContainer.style.display = 'none';
+  });
+
+  div.appendChild(img);
+  div.appendChild(links);
+  divContainer.append(div);
+  home.appendChild(divContainer);
+
+  for (let i = 0; i < linksChilds.length; i += 1) {
+    const eltChildren = linksChilds[i].children;
+    eltChildren[0].addEventListener('click', () => {
+      divContainer.style.display = 'none';
+    });
+    linksChilds[i].style.marginBottom = '20px';
+    Object.assign(eltChildren[0].style, property);
+  }
+});
+
+const speakers = [
+  {
+    img: 'statics/images/speaker_01.png',
+    name: 'Andy Jassy',
+    profession: 'CEO,Amazon',
```

Comment:
- [ ] please, kindly reduce the space between your speakers in the mobile version
- [ ] also reduce the space above the featured speakers heading to follow the original design. Nice work
yours
![Screenshot (738)_LI](https://user-images.githubusercontent.com/65298048/211083510-a314eeb7-3c43-4c2b-9b80-92c4044ee4dc.jpg)
behance
![Screenshot (737)](https://user-images.githubusercontent.com/65298048/211083632-c99b4fb9-dffd-4edb-a44c-136e40c11923.png)

Reply:
<img width="407" alt="image" src="https://user-images.githubusercontent.com/39811497/211099485-888ced97-13d4-409a-a000-dd560da924e7.png">

tried to resolve it. Thanks